docs/internal-documentation/maintenance/Garbage-Collection-Testing.md

```diff
@@ -0,0 +1,122 @@
+# Testing instructions for the Garbage Collection feature of the Zonemaster Backend module
+
+## Introduction
+The purpose of this instruction is to serve as a notice for manual testing of the new garbage collection feature at release time.
+
+## Testing the unfinished tests garbage collection feature
+
+1. Ensure that the database has the required aditionnal columns for this feature:
+```
+SELECT nb_retries FROM test_results LIMIT 0;
+```
+Should return:
+
+```
+ nb_retries 
+------------
+(0 rows)
+```
+
+2. Check that your backend_config.ini has the proper parameter set
+
+Either disabled:
+```
+#maximal_number_of_retries=3
+```
+or set to 0:
+```
+maximal_number_of_retries=0
+```
+
+3. Start a test and wait for it to be finished
+
+```
+SELECT hash_id, progress FROM test_results LIMIT 1;
+```
+Should return:
+
+```
+     hash_id      | progress 
+------------------+----------
+ 3f7a604683efaf93 |      100
+(1 row)
+```
+
+4. Sumulate a crashed test
```

Comment:
"Sumulate" -> "Simulate"

There's another one like this further down.

Reply:
Fixed in the commit 65f5257